source/server-discovery-and-monitoring/tests/README.rst

```diff
@@ -114,9 +129,14 @@ events published during client construction.
 Test Phases
 ~~~~~~~~~~~
 
-For each phase in the file, parse the "responses" array.
-Pass in the responses in order to the driver code.
-If a response is the empty object `{}`, simulate a network error.
+For each phase in the file:
+
+#. Parse the "responses" array. Pass in the responses in order to the driver
+   code. If a response is the empty object `{}`, simulate a network error.
+
+#. Parse the "applicationErrors" array. For each element, simulate the error
+   on an application connection. If a response is the empty object `{}`,
```

Comment:
I do not see "application connection" being defined in https://github.com/mongodb/specifications/tree/master/source/server-discovery-and-monitoring/tests, is this a new concept and if yes can it be defined?

Reply:
Perhaps "pooled connection"?